src/components/editor/inputs/InputLang.jsx

```diff
@@ -112,13 +118,17 @@ const InputLang = (props) => {
             </div>
           </div>
           <div className="modal-footer">
+            <span className={alertClasses.join(" ")}>
+              Please select a valid language.
+            </span>
             <button className="btn btn-link" onClick={close}>
```

Comment:
this alert will be conditionally shown if the user tries to submit the form in a bad state (we also disable the submit button until they correct the issue)